Base/res/js/SpreadSheet/runtime.js

```diff
@@ -0,0 +1,190 @@
+const sheet = this
```

Comment:
Can we please put `Libraries/LibJS/Tests/.prettierrc` into the repository root and reformat this file accordingly for consistency?

Reply:
also can we maybe stick to `camelCase` in JavaScript land, the mixture of `camelCase` built-ins and `snake_case` additional functions looks strange

Reply:
Sure.
let's see if I can't install prettier.

the naming thing poses an interesting question, do we want to keep the javascript-esque style in the spreadsheet expressions as well? should, for instance, `sumif` be `sumIf` to comply with js naming standards - which I have no idea about?